Dockerfile

```diff
@@ -0,0 +1,24 @@
+# renovate: datasource=docker depName=php
+ARG PHP_VERSION=8.3.2
+
+# renovate: datasource=docker depName=alpine
+ARG ALPINE_VERSION=3.18
+
+FROM php:${PHP_VERSION}-fpm-alpine${ALPINE_VERSION}
+
+# php extensions installer: https://github.com/mlocati/docker-php-extension-installer
+COPY --from=mlocati/php-extension-installer --link /usr/bin/install-php-extensions /usr/local/bin/
+
+RUN set -eux; \
+	install-php-extensions pdo_mysql
+
+# https://getcomposer.org/doc/03-cli.md#composer-allow-superuser
+ENV PATH="${PATH}:/root/.composer/vendor/bin"
+ENV COMPOSER_HOME=/tmp/composer
+ENV COMPOSER_CACHE_DIR=/tmp/composer/cache
+RUN mkdir -p /tmp/composer/cache
+RUN chmod ugo+w /tmp/composer/cache
+
+COPY --from=composer:2.7 /usr/bin/composer /usr/bin/composer
```

Comment:
is it required to specify the composer version there? I think picking the latest would be preferable

Reply:
No, at https://github.com/ecamp/ecamp3/blob/eaeb9c2fcbcaaeb63ba21f95662f7943ad51759f/api/Dockerfile
we use a pinned composer version, and let renovate update it.
Like this it uses the last :latest composer image you pulled.

But i changed it, this can be optimized later.